testdata/sqllogictests/xlsx.slt

```diff
@@ -57,3 +57,37 @@ select "HEADING" from read_excel('file://${PWD}/testdata/xlsx/multiple_sheets.xl
 # negatives are not allowed for infer_rows
 statement error
 select * from read_excel('file://${PWD}/testdata/xlsx/multiple_sheets.xlsx', sheet_name => 'other', infer_rows => -1);
+
+
+# create external table
```

Comment:
can we try tests/external tables with different sheet names and make sure that we get different sheets when specified?)